docs/QuestionsIssuesPrs.md

```diff
@@ -103,7 +152,38 @@ license as that used by the project.
     1. Sync your fork with upstream repo.
     1. Push the new commits to the same branch in your fork. GitHub will update the PR with your new code automatically. 
     1. When the code is ready for review again, post a comment in the PR thread to let the reviewer know.
+
+1. After reach round of review, the dev _updates existing commits_ or adds new ones in response to review comments.  
+   
+   <details>
+   <summary>Guidelines for updating the PR in response to a review</summary>
+   
+   * Updates to existing logical changes should be done by 
+     [modifying their corresponding commit](http://sethrobertson.github.io/GitPostProduction/gpp.html#analyze).
```

Comment:
I fully read through http://sethrobertson.github.io/GitPostProduction/gpp.html#analyze this time, and I think it's not a very good explanation of how to edit a commit since it focuses more on ensuring that all commits in a PR passes tests rather than explaining how to edit a *specific* commit.

The best I could find via a quick search is http://stackoverflow.com/a/1186549